### PR TITLE
143595731 kh user roles

### PR DESCRIPTION
**Pull request by kheppenstall**
-Add enum for `role` on user that can be 'student' (0) or 'staff' (1)
-Update `CensusService` to retrieve role and cohort data
-Update `UsersController` to add cohort and role after save
-Update `User` model to have `add_cohort_and_role` method that adds them appropriately based on the data retrieved from the `CensusService`
-Update rake task to update cohort and role (task is now `rake census:update`)

@Dpalazzari @wlffann @lucyconklin This is a bigger one so maybe two reviewers.

@case-eee FYI

### CODE REVIEW

**Comment by Dpalazzari** [comment]
I think this is good and okay to merge. @lucyconklin and @wlffann wanna take a look?

File: lib/tasks/census.rake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 namespace :census do
   desc "Update cohort on all users with census data"
-  task update_cohorts: :environment do
+  task update: :environment do
     User.all.each do |user|
-      user.add_cohort
+      user.add_cohort_and_role
```
Comment (Dpalazzari) [inline lib/tasks/census.rake]: We will have to set up a heroku scheduler to run this for us. @kheppenstall

Comment (kheppenstall) [inline lib/tasks/census.rake]: @Dpalazzari Yea I think so. I was hoping to run it with the update for the token so was going to wait to make a combined rake task that ran both and schedule that one.

Comment (case-eee) [inline lib/tasks/census.rake]: 👍

**Pull request merged**